src/Umbraco.Abstractions/IBackOfficeInfo.cs

```diff
@@ -0,0 +1,7 @@
+namespace Umbraco.Core
+{
+    public interface IBackOfficeInfo
+    {
+        string GetAbsoluteUrl { get; }
```

Comment:
Prob need some comments as to what this actually is: It's the absolute URL for the back office which is used in email notifications and to build URLs for back office links.

Side note - I think with all of these changes we can now remove `IServerRegistrar.GetCurrentServerUmbracoApplicationUrl` ?

Reply:
Your right, `IServerRegistrar.GetCurrentServerUmbracoApplicationUrl` is not used anymore..